spec/spec_helper.rb

```diff
@@ -17,6 +17,8 @@
 
 require "rspec/rails"
 require "webmock/rspec"
+require "rspec/json_expectations"
+
```

Comment:
Please remove this newline (rubocop thinks the newline below is the "extra blank line" ... so pronto doesn't care ;) I don't care if you delete line 21 or line 22 ;) Just make rubocop happy :) ).